packages/pwa-kit-runtime/package.json

```diff
@@ -48,9 +48,11 @@
   },
   "devDependencies": {
     "aws-lambda-test-utils": "^1.3.0",
+    "fs-extra": "^10.0.1",
     "internal-lib-build": "^2.0.0-dev.2",
     "nock": "^13.1.1",
     "node-fetch": "2.6.1",
+    "pwa-kit-build": "^2.0.0-dev.2",
```

Comment:
Adding this causes our check-dependencies.js to output an error: `Package "pwa-kit-runtime" depends on package "pwa-kit-build", but is missing one of its peerDependencies "pwa-kit-runtime@^2.0.0-dev.2"`.

Now both the runtime and build packages have peer dependency of each other. Is that all right? Or just one of them is enough..

Reply:
This is fixed.